app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/utils/rules.ts

```diff
@@ -96,3 +99,22 @@ export const checkIsRuleGroupDisabled = (allRecordsMap: Record<string, StorageRe
     return true;
   } else return false;
 };
+
+export const getAllRulesOfGroup = (appMode: AppMode, groupId: string): Promise<Rule[]> => {
+  if (!groupId) return Promise.resolve([]);
+
+  return new Promise<Rule[]>((resolve) => {
+    StorageService(appMode)
+      .getAllRecords()
+      .then((allRecords) => {
+        const groupRules = Object.values(allRecords).filter(
+          (record: StorageRecord) => isRule(record) && record?.groupId === groupId
+        );
+        resolve(groupRules as Rule[]);
+      })
+      .catch((error) => {
+        Logger.log(error);
+        resolve([]);
+      });
+  });
+};
```

Comment:
I don't think there is one specific file for such utils, but I believe it would make more sense to move it to [`src/utils/rules/misc.js`](https://github.com/requestly/requestly/blob/48cd10fb7eb2feb95787c217be1f7d68f59711b8/app/src/utils/rules/misc.js)